db/views/patient_registrations_per_day_per_facilities_v01.sql

```diff
@@ -1,11 +1,11 @@
 SELECT COUNT(patients.id) AS registration_count,
        patients.registration_facility_id AS facility_id,
-       patients.deleted_at at time zone 'utc' at time zone (SELECT current_setting('TIMEZONE')) AS deleted_at,
-       patients.recorded_at at time zone 'utc' at time zone (SELECT current_setting('TIMEZONE')) AS recorded_at,
+       facilities.deleted_at at time zone 'utc' at time zone (SELECT current_setting('TIMEZONE')) AS deleted_at,
 cast(EXTRACT(DOY FROM patients.recorded_at at time zone 'utc' at time zone (SELECT current_setting('TIMEZONE'))) as text) AS day,
 cast(EXTRACT(MONTH FROM patients.recorded_at at time zone 'utc' at time zone (SELECT current_setting('TIMEZONE'))) as text) AS month,
 cast(EXTRACT(QUARTER FROM patients.recorded_at at time zone 'utc' at time zone (SELECT current_setting('TIMEZONE'))) as text) AS quarter,
 cast(EXTRACT(YEAR FROM patients.recorded_at at time zone 'utc' at time zone (SELECT current_setting('TIMEZONE'))) as text) AS year
 FROM patients
+INNER JOIN facilities ON patients.registration_facility_id = facilities.id
 WHERE patients.deleted_at IS NULL
-GROUP BY day, month, quarter, year, facility_id, patients.deleted_at, patients.recorded_at;
+GROUP BY day, month, quarter, year, facility_id, facilities.deleted_at;
```

Comment:
I don't think we need to group by `facilities.deleted_at`

Reply:
The `deleted_at` needs to be grouped by since it's part of the `SELECT` clause. We need the `deleted_at` for activerecord to apply the `kept` scope.